client/my-sites/site-settings/form-general.jsx

```diff
@@ -700,6 +711,7 @@ const getFormSettings = ( settings ) => {
 		wpcom_locked_mode: false,
 		wpcom_public_coming_soon: '',
 		wpcom_gifting_subscription: false,
+		wpcom_admin_interface: 'calypso',
```

Comment:
I think this change is not necessary (same as line 736 below). Can you try removing them?

Reply:
Removed by https://github.com/Automattic/wp-calypso/pull/88459/commits/06ffd32728890dedb01e378d6cb57058fb1cde47.